perllib/FixMyStreet/SendReport/Open311.pm

```diff
@@ -127,7 +127,7 @@ sub send {
             $revert = 1;
         }
 
-        if ($row->cobrand eq 'fixmybarangay') {
+        if ($row->cobrand eq 'fixmybarangay' || $row->cobrand eq 'greenwich') {
             # FixMyBarangay endpoints expect external_id as an attribute, as do Oxfordshire
```

Comment:
That's a good point. It seems that Oxfordshire do require `external_id`, but it's handled elsewhere so I don't know why this comment mentions Oxfordshire. I'll update it to say Greenwich instead.